oak/server/storage/storage_node.h

```diff
@@ -32,9 +36,21 @@ class StorageNode final : public NodeThread {
   StorageNode(const std::string& name, const std::string& storage_address);
 
  private:
+  const asylo::StatusOr<std::string> Encrypt(asylo::AesGcmSivCryptor* cryptor,
+                                             const absl::string_view& data);
+  const asylo::StatusOr<std::string> EncryptDatumName(const absl::string_view& datum_name);
+  const asylo::StatusOr<std::string> EncryptDatumValue(const absl::string_view& datum_value);
```

Comment:
I *think* that `EncryptDatumValue` requires there to have been an earlier call to `EncryptDatumName` to set up the nonce; if so, could do with a comment here.

Reply:
The name and value use different nonces.  Restructured and attempted to clarify.